newsletter/models.py

```diff
@@ -3,8 +3,12 @@
 
 # Create your models here.
 class SubscribedUser(models.Model):
-    name = models.CharField(max_length=30)
     email = models.EmailField()
 
     def __str__(self):
         return str(self.email)
+
+    class Meta:
```

Comment:
You can use this to simplify the model. It has the same effect: https://docs.djangoproject.com/en/4.0/ref/models/fields/#unique